### DISCUSSION TITLE
Welcome to MimeKit Discussions!

### DISCUSSION THREAD

**Discussion by jstedfast** [jstedfast/MimeKit#953] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abrguyt** [comment]
Using MailKit/MimeKit happily and thankful for its advanced functions like Builder, DKIM signing, IMAP OAuth access, etc. Impressed with the richness of the suite.

Recently I noticed that when preparing a mail with a MimeMessage with 'message.Prepare(EncodingConstraint.SevenBit)' and the last line is longer that the default max nr of characters per line (78) **and** the line does **not** end with a newline (\n\r), MimeKit seems to erroneously add a '=' symbol (which is the Quoted-Printable soft line-break symbol) to that last line in the encoded bytestram - which then shows up in the receiving email client. Adding a simple newline (\n\n) to that last line solves the problem, but I don't think that it's how it supposed to work.

Anyone else experiencing this? (using latest version 4.7.0 of MimeKit)

**Reply by jstedfast** [reply]
Thanks, can you submit a bug report about this?

**Reply by abrguyt** [reply]
done .. thanks.
